.github/workflows/image-retention.yaml

```diff
@@ -7,6 +7,9 @@ on:
     branches:
       - "*"
       - "!main" # excludes main
+    paths:
```

Comment:
Thanks, that was much needed. Do we still need the branch filter to exclude main with the path filter set?

Reply:
Nice hint. I'll tried it out. When no branch-filter matches the pipeline does not run at all.